synapse/models/science.py

```diff
@@ -0,0 +1,92 @@
+import synapse.lib.module as s_module
+
+class ScienceModule(s_module.CoreModule):
+
+    def getModelDefs(self):
+        return (('sci', {
+            'types': (
+                ('sci:hypothesis:type:taxonomy', {}, {}),
+                ('sci:hypothesis', ('guid', {}), {
+                    'doc': 'A hypothesis or theory.'}),
+
+                # TODO link experiment to eventual procedure node
+                ('sci:experiment:type:taxonomy', {}, {}),
+                ('sci:experiment', ('guid', {}), {
+                    'doc': 'An instance of running an experiment to test a hypothesis.'}),
+
+                ('sci:observation', ('guid', {}), {
+                    'doc': 'An observation which may have resulted from an experiment.'}),
+
+                ('sci:evidence', ('guid', {}), {
+                    'doc': 'An assessment of how an observation supports or refutes a hypothesis.'}),
+            ),
+
+            'edges': (
+                (('sci:experiment', 'uses', None), {
+                    'doc': 'The experiment used the target nodes when it was run.'}),
+                (('sci:observation', 'has', None), {
+                    'doc': 'The observations are summarized from the target nodes.'}),
+                (('sci:evidence', 'has', None), {
+                    'doc': 'The evidence includes observations from the target nodes.'}),
+            ),
+
+            'forms': (
+                # TODO many of these forms need author/contact props
+                ('sci:hypothesis:type:taxonomy', {}, {}),
+                ('sci:hypothesis', {}, (
```

Comment:
Do we want to give analysts the ability to link related hypothesis, like ACH? If so do we want to add an :ach property or something of the like?

Reply:
Definitely not an ACH property since it would need to be multiple. We could consider some type of mutual exclusion between hypotheses eventually, but definitely not for this iteration.